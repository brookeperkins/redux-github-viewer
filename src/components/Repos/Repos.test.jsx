import React from 'react';
import { render, cleanup } from '@testing-library/react';
import RepoList from './Repos';
import { Provider } from 'react-redux';
import store from '../../store';

describe('Repos component', () => {
  afterEach(() => cleanup());
  it('renders Repos', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <RepoList />
      </Provider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
