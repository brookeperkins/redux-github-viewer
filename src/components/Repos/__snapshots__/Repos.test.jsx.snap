// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repos component renders Repos 1`] = `
<DocumentFragment>
  <ul
    data-testid="repo-list"
  />
</DocumentFragment>
`;
